pkg/image/api/helper.go

```diff
@@ -463,6 +463,8 @@ func ImageWithMetadata(image *Image) error {
 			return nil
 		}
 
+		image.DockerImageManifestMediaType = schema1.MediaTypeManifest
```

Comment:
We'll need the migrator in order to make this work then?  How will a user upgrading get the correct behavior?  Or is this just extra data we forgot?